docs/UC_CV_3ElectrodeCell.md

```diff
@@ -0,0 +1,40 @@
+Use Case: Cyclic Voltammetry on the JUAMI Potentiostat
+Intended User: Undergraduate Researcher at a University in Africa with Introductory Knowledge of Electrochemistry and Cyclic Voltammetry. The instructions in this UC assume that the student already has an electrochemical cell set up and ready for testing (working electrode, counter electrode, and possibly reference electrode are all in a proper beaker).
+As a test case - Mike will perform these instructions as someone with knowledge of electrochemistry but little knowledge of Python.
+
+Experimental Setup:
+
+1. Mike plugs the USB cable into the computer and the potentiostat
+1. Mike attaches the red wire to the working electrode (material of interest)
+1. Mike attaches the black clip to the counter electrode
+1. Mike attaches the white clip to the reference electrode if he is using one, or he attaches it to the counter electrode for a two electrode test
+
+Program Setup:
+1. Mike opens the Pytentiostat script (the script is incomplete and requires filling in of user-defined information)
+1. Mike types a file name for the sample being tested next to 'file name' on line # (file will contain the voltage, current, and time information)
+1. Mike inputs Initial Potential on line # (V vs. open circuit voltage (Eocv))
```

Comment:
How will a new user know what value to type here?  I guess that if it is a new user they will be following a tutorial which will tell them.  In that case we need a step in there which is 
1. Mike opens the ``Getting Started`` documentation and follows the steps, sthg like that.